.github/workflows/ci.yml

```diff
@@ -173,6 +173,12 @@ jobs:
         run: |
           xvfb-run make -C doc html SPHINXOPTS="-j auto -W --keep-going"
 
+      - name: "Substitute defective GIF"
+        run: |
+          cd doc/_build/html/examples/gallery_examples/00-mapdl-examples
+          cp ../../../../../source/images/dcb.gif ../../../_images/
+          sed -i 's+../../../_images/sphx_glr_composite_dcb_004.gif+../../../_images/dcb.gif+g' composite_dcb.html
+
```

Comment:
Do we need to go back to the working directory after issuing ``cd``??

Reply:
I don't think it is needed as it was working fine, but added a cd to initial folder just in case.